simulatingrisk/hawkdovemulti/batch_run.py

```diff
@@ -178,7 +212,9 @@ def main():
         args.processes,
         args.max_steps,
         args.progress,
+        args.agent_data,
         args.file_prefix,
+        args.max_runs,
```

Comment:
I feel like the number of parameters is getting a little excessive but I'm hesitant to tie too closely to argparse opts. Any suggestions?